features/flows/rest/api/src/main/java/org/opennms/netmgt/flows/rest/classification/ClassificationRestService.java

```diff
@@ -72,6 +73,10 @@
     @Path("{id}")
     Response updateRule(@PathParam("id") int id, RuleDTO newValue);
 
+    @PATCH
+    @Path("{id}")
+    Response patchPositionOfRule(int id, RuleDTO patch);
```

Comment:
The `@PathParam("id")` annotation is missing.

Looking at this, it suggest to be identical to `@PUT`, but it is not.
I'd say either : 
 - use `updateRule()`, 
 - or only send the position instead of the whole object.
 - Alternatively fully implement a `patchRule()` method.

I'd opt for the 2nd option.

Reply:
we will use PUT instead